### PR TITLE
standardize atomic physics stage signatures

### PR DESCRIPTION
**Pull request by BrianMarre**
standardizes the method signatures of the AtomicPhyiscs(FLYonPIC) stage

- [x] requires PR #5194 to be merged first
- [x] must be rebased to dev

### CODE REVIEW

File: include/picongpu/simulation/stage/AtomicPhysics.x.cpp [hunk 1]
```diff
@@ -107,7 +107,7 @@ namespace picongpu::simulation::stage
                 = particles::atomicPhysics::localHelperFields::FoundUnboundIonField<picongpu::MappingDesc>;
 
             // species Lists
-            //{
+            //@{
```
Comment (psychocoderHPC) [inline include/picongpu/simulation/stage/AtomicPhysics.x.cpp]: only a note: This and the line above must be a doxygen comment e.g. `///` or `//!` or `/**`

**Review by psychocoderHPC — DISMISSED** [review]

**Comment by psychocoderHPC** [comment]
I am fine with this PR even without rebasing. Two commits can be checked very easy

**Review by psychocoderHPC — DISMISSED** [review]

**Comment by psychocoderHPC** [comment]
@BrianMarre OK github likes to see a rebase because it says the mergebase changed. Please rebase.

File: include/picongpu/simulation/stage/AtomicPhysics.x.cpp (hunk 1 above)
Comment (BrianMarre) [reply include/picongpu/simulation/stage/AtomicPhysics.x.cpp]: fixed

**Review by psychocoderHPC — APPROVED** [review]

**Pull request merged**